pyeti/api.py

```diff
@@ -231,6 +231,45 @@ def observable_bulk_add(self, observables, tags=None):
         json = {"observables": [{"tags": tags, "value": o} for o in observables]}
         return self._make_post('observable/bulk', json=json)
 
+    def launch_analytics_oneshot(self, name_of_oneshot, value, type_obversable):
```

Comment:
I know this is bad, but please prepend the function name by the thing you're working on and ideally the HTTP endpoint that will be called, e.g. `analytics_oneshot_run`